runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/ExecutionStateSampler.java

```diff
@@ -68,10 +68,15 @@ public static ExecutionStateSampler newForTest(MillisProvider clock) {
     return new ExecutionStateSampler(checkNotNull(clock));
   }
 
-  private static final long PERIOD_MS = 200;
+  private static long periodMs = 200;
```

Comment:
Please add a comment talking about this value might be reset via experiments pipeline options.

Reply:
done.